### ISSUE TITLE
Hotplugging CPU and RAM

### ISSUE THREAD

**Issue by bt-josh** [nutanix/terraform-provider-nutanix#79]
**Describe the bug**
Hotplugging for CPU and RAM appears to have been enabled in #75 however VMs are still rebooting after increasing either `num_vcpus_per_socket` or `memory_size_mib`.

**Expected behavior**
Number of CPUs or Memory should increase without a system restart.

**Logs**
N/A

**Versions (please complete the following information):**
 - OS: RHEL 7
 - Terraform 0.12.8
 - Connecting to: Prism Element 5.9
 - Not connecting to Prism Central
 - Nutanix Terraform Provider v1.0.2

**Additional context**
We don't currently use the Prism Central for our Terraform connections, we connect directly to the Prism Element instead.

**Comment by Jorge-Holgado** [comment]
Good morning,
I didn't patch tf 0.12. Don't know if @marinsalinas  has ported the change to 0.12 branch.
Maybe he can solve our doubts.

**Comment by bt-josh** [comment]
The TF 0.12 code was merged into master as part of the same provider release (v1.0.2) as the hotplug code, so I don't think there's a discrete 0.12 branch anymore.

**Comment by mritzmann** [comment]
I found this commit from @Jorge-Holgado and @marinsalinas: https://github.com/terraform-providers/terraform-provider-nutanix/commit/f61aa7364b0c15aa069e82846e5565771141a1db

```
Patch to allow some changes to be hotplug

Options that will be hot-plug (always):
  * name
  * description

Options hot-plug just when increasing it:
  * num_vcpus_per_socket
  * num_sockets
  * memory_size_mib
```

In my case, however, I explicitly want a reboot, since my guest system does not support hotplug. Is there a way to force a restart by terraform? Is there a way to disable hot-plug?

**Comment by coderGo93** [comment]
Hello @bt-josh  , I made [PR](https://github.com/terraform-providers/terraform-provider-nutanix/pull/136) with branch `fix-86/79` , a new parameter to force reboot for Cpus/memory changes so in your case it shouldn't reboot unless asked, so if it's possible you can try it and see if it's ok, please?

**Comment by coderGo93** [comment]
Hello, I'm going to close because there are not responses from the issuer and it has been added a parameter for this case and merged to master and applies for #86 too.
